stories/adminbar.stories.js

```diff
@@ -21,57 +21,905 @@
+				},
+				{
+					clicks: 0,
+					ctr: 0,
+					impressions: 64,
+					keys: [
+						'2021-01-14',
+					],
+					position: 2.625,
+				},
+				{
+					clicks: 0,
+					ctr: 0,
+					impressions: 33,
+					keys: [
+						'2021-01-15',
+					],
+					position: 2.606060606060606,
+				},
+				{
+					clicks: 0,
+					ctr: 0,
+					impressions: 49,
+					keys: [
+						'2021-01-16',
+					],
+					position: 2.63265306122449,
+				},
+				{
+					clicks: 0,
+					ctr: 0,
+					impressions: 21,
+					keys: [
+						'2021-01-17',
+					],
+					position: 2.4761904761904763,
+				},
+				{
+					clicks: 0,
+					ctr: 0,
+					impressions: 50,
+					keys: [
+						'2021-01-18',
+					],
+					position: 2.62,
+				},
+				{
+					clicks: 2,
+					ctr: 0.03571428571428571,
+					impressions: 56,
+					keys: [
+						'2021-01-19',
+					],
+					position: 4,
+				},
+				{
+					clicks: 0,
+					ctr: 0,
+					impressions: 52,
+					keys: [
+						'2021-01-20',
+					],
+					position: 2.6346153846153846,
+				},
+				{
+					clicks: 0,
+					ctr: 0,
+					impressions: 63,
+					keys: [
+						'2021-01-21',
+					],
+					position: 2.4603174603174605,
+				},
+				{
+					clicks: 1,
+					ctr: 0.02040816326530612,
+					impressions: 49,
+					keys: [
+						'2021-01-22',
+					],
+					position: 2.2653061224489797,
+				},
+				{
+					clicks: 0,
+					ctr: 0,
+					impressions: 15,
+					keys: [
+						'2021-01-23',
+					],
+					position: 2.7333333333333334,
+				},
+				{
+					clicks: 1,
+					ctr: 0.03571428571428571,
+					impressions: 28,
+					keys: [
+						'2021-01-24',
+					],
+					position: 2.571428571428571,
+				},
+				{
+					clicks: 2,
+					ctr: 0.04,
+					impressions: 50,
+					keys: [
+						'2021-01-25',
+					],
+					position: 5.84,
+				},
+				{
+					clicks: 0,
+					ctr: 0,
+					impressions: 56,
+					keys: [
+						'2021-01-26',
+					],
+					position: 8.303571428571429,
+				},
+				{
+					clicks: 0,
+					ctr: 0,
+					impressions: 39,
+					keys: [
+						'2021-01-27',
+					],
+					position: 2.564102564102564,
+				},
+			];
+			// "options": {
+			// 	"startDate": "2020-12-03",
+			// 	"endDate": "2021-01-27",
+			// 	"dimensions": "date",
+			// 	"url": "https://www.elasticpress.io/features/"
+			// }
+			const searchConsoleOptions = {
+				startDate: '2020-12-03',
+				endDate: '2021-01-27',
+				dimensions: 'date',
+				url: 'https://www.sitekitbygoogle.com/blog/',
+			};
 
-		removeAllFilters( 'googlesitekit.AdminbarModules' );
-		addFilter( 'googlesitekit.AdminbarModules',
-			'googlesitekit.Analytics',
-			addAnalyticsAdminbarWidget, 11 );
+			registry.dispatch( MODULES_SEARCH_CONSOLE ).receiveGetReport( searchConsoleData, { options: searchConsoleOptions } );
 
-		addFilter( 'googlesitekit.AdminbarModules',
-			'googlesitekit.SearchConsole',
-			addGoogleSitekitSearchConsoleAdminbarWidget );
+			// Mock Analytics mock data separately for each section
+			// Total Users
+			const analyticsTotalUsersData = [
```

Comment:
There's a lot of non-dynamic data in this file, can we move them out into fixtures/a separate file? It'd help if we ever wanna reuse it again, but mostly this data makes this file quite long and makes it hard to read the actual story logic.

Reply:
Moved to fixtures.